src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -46,10 +46,8 @@ public function __construct( $args = array() ) {
 			)
 		);
 
-		$allowed_statuses = array( 'active', 'inactive', 'recently_activated', 'upgrade', 'mustuse', 'dropins', 'search', 'paused', 'auto-update-enabled', 'auto-update-disabled' );
-
 		$status = 'all';
-		if ( isset( $_REQUEST['plugin_status'] ) && in_array( $_REQUEST['plugin_status'], $allowed_statuses, true ) ) {
+		if ( isset( $_REQUEST['plugin_status'] ) ) {
 			$status = $_REQUEST['plugin_status'];
```

Comment:
```suggestion
			$status = sanitize_text_field( $_REQUEST['plugin_status'] );
```

Since we removed the hardcoded status check, we need to sanitize the one we get from `$_REQUEST`.